//multiple try catch block ontry block having multiple catch block
public class Exception_example_2 {

	public static void main(String[] args) {
		try {
			int StudentCount[] = new int[20]; //of a particular class
			int Percentage_width;  //percentage width of a class
 			StudentCount[10]=30;
 			Percentage_width=StudentCount[10]/0;
		}
		
		catch(ArithmeticException e){
			System.out.println("Arithmetic Exception occurs");	
		}
		
		catch(ArrayIndexOutOfBoundsException e)  
        {  
         System.out.println("ArrayIndexOutOfBounds Exception occurs");
	    }
		
		finally{
			System.out.println("Class Width cant be zero");
			System.out.println("code completed ");
		}

}
}	
